adapters/THIRD_PARTY_LICENSES.md

```
@@ -0,0 +1,29 @@
+# Third-Party Licenses
+
+`modbusadapter` and `dummymodbusadapter` statically link the following third-party library. Its
+license text must accompany any distributed copy of these binaries.
+
+## libsodium
+
+- Project: <https://libsodium.org>
+- License: ISC License
+- Copyright: Copyright (c) 2013-2026 Frank Denis <j at pureftpd dot org>
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)